governance/cla/index.html

```diff
@@ -1,3 +1,3 @@
 ---
-redirect_to: "./cla.pdf"
+redirect_to: "./governance/cla/cla.pdf"
```

Comment:
Should lose the initial `.` to remove ambiguity.

Reply:
New PR #3 created to resolve that. Good suggestion.